feat(content): New descriptions for Remnant and Wanderer Licenses (#11373)
endless-sky/endless-sky: data/remnant/remnant outfits.txt

	series "Licenses"
	index 03010
	thumbnail "outfit/remnant license"

> 	description "In addition to serving as a handy IFF verifier, personal identification tool, and license to purchase Remnant technology, this device features movable parts that mechanically click with every adjustment. When the pieces are switched in a specific order and rhythm, the series of clicks produce a surprisingly beautiful blend of classical music tracks."

Reviewer: Rereading this - a series technically isn't plural so produce should be produce**s**.

```suggestion
	description "In addition to serving as a handy IFF verifier, personal identification tool, and license to purchase Remnant technology, this device features movable parts that mechanically click with every adjustment. When the pieces are switched in a specific order and rhythm, the series of clicks produces a surprisingly beautiful blend of classical music tracks."
```

> 	description "In addition to serving as a handy IFF verifier, personal identification tool, and license to purchase Remnant technology, this device features movable parts that mechanically click with every adjustment. When the pieces are switched in a specific order and rhythm, the series of clicks produces a surprisingly beautiful blend of classical music tracks."

Reviewer 2: Just one question: do the Remnant pay attention to "classical" music? If they've been cut off from ancestral space for such a long time, would the player know their songs?
```suggestion
	description "In addition to serving as a handy IFF verifier, personal identification tool, and license to purchase Remnant technology, this device features movable parts that mechanically click with every adjustment. When the pieces are switched in a specific order and rhythm, the series of clicks produces a surprisingly beautiful blend of music tracks."
```
I think removing it altogether would be the best way to fix the discrepancy.

Author: Aventine's spaceport: `The oldest buildings in the Remnant capital city harken back to classic human architecture, with stone facades and columns reminiscent of ancient Rome. But as you walk outward toward the more recent additions, the buildings become less and less recognizably human, with curved organic shapes and hundreds of overhanging walkways and balconies. It is as if the layers left behind by centuries of habitation are a frozen record of the slow transition of Remnant culture into something bizarre and almost alien.`

Since the player knows of ancient Rome, and the Remnant seemed to have an inspiration in human antiquity at first at least, I don't think it too far-fetched to have them know of classical music. If it's better to have it not specified, can remove the word anyhow.

Reviewer 2: Hmm, it's probably fine than.

> 	description "In addition to serving as a handy IFF verification and personal identification tool, this object consists of several movable parts, with every move triggering a mechanical click. If one switches the pieces' positions and orientation in a specific order, in the right rhythm, the series of clicks produced is a surprisingly beautiful blend of classical music tracks."

Reviewer 3: ```suggestion
	description "In addition to serving as a handy IFF verification and personal identification tool, this device consists of several movable parts, with every move triggering a mechanical click. If one switches the pieces' positions and orientation in a specific order, in the right rhythm, the series of clicks produced is a surprisingly beautiful blend of classical music tracks."
```

No reason not to be more specific.

outfit "Remnant Capital License"
	category "Licenses"
	series "Licenses"
	index 03020
	thumbnail "outfit/remnant capital license"

> 	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum, the frequency of which can carry over vast distances while on a planet, allowing captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid, should they ever become stranded."

Reviewer 2: ```suggestion
	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum, the frequency of which can carry over vast distances while on a planet, allowing captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid should they ever become stranded."
```

> 	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum, the frequency of which can carry over vast distances while on a planet. This humming allows captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid should they ever become stranded."

Reviewer 2: Simplification.
```suggestion
	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum which can carry over vast distances on a planet. This humming allows captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid should they ever become stranded."
```

> 	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum, the frequency of which can carry over vast distances while on a planet, allowing captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid should they ever become stranded."

Reviewer 3: The second sentence is a bit of a run-on. Maybe:

```suggestion
	description "While seemingly innocuous at first, this device shows the brilliant simplicity of Remnant engineering. Tapping either edge produces a brief, faint hum, the frequency of which can carry over vast distances while on a planet. This humming allows captains who have been entrusted with the Remnant's most valuable technology to call nearby patrols for aid should they ever become stranded."
```
